### PR TITLE
add mo2

### PR DESCRIPTION
**Pull request by iovw**
#91

### CODE REVIEW

**Comment by iovw** [comment]
Is this ok now? @Calinou

File: bucket/mo2.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://github.com/ModOrganizer2/modorganizer",
+    "description": "Mod manager for various PC games",
+    "version": "2.2.2.1",
+    "license": "GPL-3.0",
+    "url": "https://github.com/ModOrganizer2/modorganizer/releases/download/v2.2.2.1/Mod.Organizer-2.2.2.1.7z",
+    "hash": "sha256:65B254431AC5F3BA23251694D8F7B8910243A412EE9D8AEDE1C3259705565FAF",
```
Comment (Calinou) [inline bucket/mo2.json]: The default hash format is SHA-256, no need to specify it. Also, we use lowercase as per convention:

```suggestion
    "hash": "65b254431ac5f3ba23251694d8f7b8910243a412ee9d8aede1c3259705565faf",
```

Comment (iovw) [reply bucket/mo2.json]: I mean I think explicit is better than implicit.

**Comment by Calinou** [comment]
Thanks!

**Pull request merged**